src/main/java/ch/njol/skript/effects/EffDetonate.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.*;
+import org.bukkit.entity.minecart.ExplosiveMinecart;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+
+@Name("Detonate Entity")
+@Description("Automatically detonates an entity.\n Accepted entities are 'Fireworks, TNT Minecarts and WindCharges")
```

Comment:
```suggestion
@Description("Automatically detonates an entity.\n Accepted entities are 'Fireworks, TNT Minecarts and WindCharges.")
```
Should end with a period.

Reply:
also, `'Fireworks` <-- what is the `'` all about?
Why the `\n`?

Reply:
The \n was meant to represent a new line, but that was just personal preference. 
The `'` was meant to go around each accepted entity, but I forgot to surround it and the others, oops

Reply:
for new lines just use `({"blah", "blah"})`
also, I don't think the `'` is needed at all